clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -754,6 +752,49 @@ public void subscribe(Pattern pattern) {
         delegate.subscribe(pattern);
     }
 
+    /**
+     * Subscribe to all topics matching specified pattern to get dynamically assigned partitions.
+     * The pattern matching will be done on the broker against topics existing at the time of check
```

Comment:
Better would be "at the time of the check and matching topics will be returned". No need to (s) which is a bit messy.